programs/drift/src/math/amm_spread.rs

```diff
@@ -456,10 +484,12 @@ pub fn calculate_spread_reserves(
         spread.cast::<i32>()?.safe_add(amm.reference_price_offset)?
     };
 
-    let quote_asset_reserve_delta = if spread_with_offset.abs() > 1 {
+    let quote_asset_reserve_delta = if spread_with_offset.abs() > 2 {
+        let quote_reserve_divisor =
+            BID_ASK_SPREAD_PRECISION_I128.safe_div(spread_with_offset.cast::<i128>()? / 2)?;
```

Comment:
think you can just do ((BID_ASK_SPREAD_PRECISION_I128 / spread_with_offset) / 2) to avoid error?